tests/e2e/test_sampling_params.py

```diff
@@ -212,6 +212,7 @@ def test_spyre_batch1_top_k(model: ModelInfo, backend, monkeypatch,
     assert token_div1 < token_div2
 
 
+@pytest.mark.parametrize("mode", ["sb", "cb", "cp"])
```

Comment:
The default parameterization uses `chunked_prefill` for the cp mark, which is what the test sorting keys off of, so this may be why tests are failing in GHA due to sorting.